staging/src/k8s.io/apiserver/hack/update-codegen.sh

```diff
@@ -0,0 +1,81 @@
+
+#!/usr/bin/env bash
```

Comment:
Why are we copying hack scripts into staging repos? That doesn't seem like a good precedent.

Reply:
I copied the pattern from the sample api server, should this be placed elsewhere?

Reply:
It looks like we already do this: https://github.com/kubernetes/kubernetes/blob/master/hack/update-codegen.sh#L133

If you follow this pattern, this script should be called from there too.

Reply:
yeah thats the pattern I went with, if we theres a new pattern were shooting for I'm happy to change

Reply:
:'(